okhttp/src/test/java/okhttp3/KotlinSourceCompatibilityTest.kt

```diff
@@ -67,6 +85,13 @@ class KotlinSourceCompatibilityTest {
     val certificatePinner: CertificatePinner? = address.certificatePinner()
   }
 
+  @Test @Ignore
```

Comment:
Are these just proving that the code compiles and is idiomatic?

Reply:
Exactly. I wanna run these against 3.14 and make sure they pass there and on 4.0.